packages/core/src/components/EditableText/__stories__/EditableText.stories.tsx

```diff
@@ -102,3 +102,25 @@ export const Types = {
     }
   }
 };
+
+const multilineValue = "This is a multiline\nhere's the second line";
+export const Multiline = {
+  render: () => (
+    <div className={styles.multilineContainer}>
+      <EditableText
+        type={EditableText.types.TEXT1}
+        weight={EditableText.weights.NORMAL}
+        multiline
+        value={multilineValue}
+        className={styles.editableText}
```

Comment:
Let's avoid using css in stories

Reply:
i am using here the same CSS class other stories here are using